src/Pages/chanhoyoon/components/Post/PostComment.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './PostComment.scss';
+
+class PostComment extends React.Component {
+
+  state = {
+    toggleLike: false
+  };
+
+  toggleLike = () => {
+    this.state.toggleLike ? this.setState({ toggleLike: false }) : this.setState({ toggleLike: true });
+  };
```

Comment:
`!` 연산자를 이용해서 더 간한하게 수정할 수 있겠네요!
```suggestion
  toggleLike = () => {
    this.setState({ toggleLike: !이전toggleLike값 })
  };
```